lib/hanami/generators/app/application.rb.tt

```diff
@@ -85,20 +94,24 @@ module <%= config[:classified_app_name] %>
       #
       # default_request_format :html
 
-      # Default format for responses that doesn't take into account the request format
+      # Default format for responses that doesn't take into account the request
+      # format
+      #
       # Argument: A symbol representation of a mime type, default to :html
       #
       # default_response_format :html
 
       # HTTP Body parsers
       # Parse non GET responses body for a specific mime type
-      # Argument: Symbol, which represent the format of the mime type (only `:json` is supported)
+      # Argument: Symbol, which represent the format of the mime type
+      #             (only `:json` is supported)
       #           Object, the parser
       #
       # body_parsers :json
 
       # When it's true and the router receives a non-encrypted request (http),
-      # it redirects to the secure equivalent resource (https). Default disabled.
+      # it redirects to the secure equivalent resource (https).
+      # Default disabled.
```

Comment:
113 and 114 can be combined into one line by rewording to:
` # it redirects to the secure equivalent (https). Disabled by default.`
(dropping 'resource' because it redirects for all routes, not just RESTful/resourceful routes)